src/components/MonitorListFilter.vue

```diff
@@ -191,52 +194,63 @@ export default {
         this.getExistingTags();
     },
     methods: {
+        getActiveFilters: function () {
+            const filters = Object.fromEntries(
+                Array.from(Object.entries(this.$router.currentRoute.value.query ?? {}))
+            );
+
+            return {
+                status: filters["status"] ? filters["status"].split(",") : [],
```

Comment:
Here you try to store the query as a comma separated string.
I think there is a simpler way, as vue-router seems to allow storing lists ⇒ should be able to retrieve it
https://stackoverflow.com/questions/50692081/vue-js-router-query-array

Is there a reason for this approach I am not seeing?

Reply:
Didn't know that existed. Very nice, will rewrite that part for tags.

Reply:
@CommanderStorm Do you want me to rewrite that for every filter? Is the current approach not fine?

Reply:
> Here you try to store the query as a comma separated string. I think there is a simpler way, as vue-router seems to allow storing lists ⇒ should be able to retrieve it https://stackoverflow.com/questions/50692081/vue-js-router-query-array
> 
> Is there a reason for this approach I am not seeing?

@CommanderStorm Do you want me to rewrite that for every filter? The current approach works well, is it not fine?

Reply:
@CommanderStorm Do you want me to rewrite that for every filter? The current approach works well, is it not fine?

Reply:
Yes, please consider using the tools provided via the vue router instead of inventing a new way of storing lists.
This should significantly clean up the code ^^